src/index.ts

```diff
@@ -85,6 +105,7 @@ function createStyle(bundle: AnserJsonEntry): Colors {
 
 function convertBundleIntoReact(
   linkify: boolean,
```

Comment:
At the risk of avoiding too many magic booleans in this API, should we make `linkify` and enum that takes different values for the new vs. old behavior?